buildfile.m

```diff
@@ -0,0 +1,13 @@
+function plan = buildfile
+
+% Add source code folder to path
+addpath("code");
+% Create a plan
+plan = buildplan();
```

Comment:
Do we want `buildplan(localfunctions)` to make it easier for people to add adhoc tasks? Is that something we always document in the usage of `buildplan`? Could go either way on this but, I prefer `buildplan(localfunctions)` to make it easier to customize later.